portfolio/src/main/java/com/google/sps/servlets/LoginAPI.java

```diff
@@ -40,39 +40,42 @@
 @WebServlet("/login-check")
 public class LoginAPI extends HttpServlet {
 
-  private static class Comment {
-    long id;
-    long timestamp;
-    String userName;
-    String comment;
+  private static class LoginInfo {
+    boolean isLoggedIn; 
+    String loginLink; 
+    String logoutLink; 
 
-    public Comment(long id, long timestamp, String userName, String comment) {
-      this.id = id;
-      this.timestamp = timestamp;
-      this.userName = userName;
-      this.comment = comment;
+    public LoginInfo(boolean isLoggedIn, String loginLink, String logoutLink) {
+      this.isLoggedIn = isLoggedIn; 
+      this.loginLink = loginLink; 
+      this.logoutLink = logoutLink; 
     }
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html");
+    response.setContentType("application/json");
 
     UserService userService = UserServiceFactory.getUserService();
-    if (userService.isUserLoggedIn()) {
-      String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/index.html"; 
-      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut); 
-      response.getWriter().println("<p>You're logged in, good job.</p>");
-      response.getWriter().println("<p>Your email is: " + userEmail + "!</p>");
-      response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>"); 
+    boolean isLoggedIn = userService.isUserLoggedIn(); 
+    String loginLink; 
```

Comment:
You could use ternary operator https://www.baeldung.com/java-ternary-operator

String loginLink = (isLoggedIn): null : userService.createLoginURL("/index.html");
String logoutLink = (isLoggedIn): userService.createLoginURL("/index.html") : null;

Reply:
Cool, I had seen that in the javascript tutorials, but I didn't know it worked for java too.